zeppelin-web/src/app/notebook/notebook.controller.js

```diff
@@ -150,7 +152,7 @@ angular.module('zeppelinWebApp').controller('NotebookCtrl', function($scope, $ro
   };
 
   $scope.killSaveTimer = function() {
```

Comment:
This should be changed to var killsaveTimer = function()
There is no point of having it in the $scope.